apollo-router/src/services/supergraph_service.rs

```diff
@@ -492,7 +492,7 @@ mod tests {
    @join__owner(graph: ORGA)
    @join__type(graph: ORGA, key: "id")
    @join__type(graph: USER, key: "id") {
-       id: ID
+       id: ID!
```

Comment:
Hmm, that seems like a material change which isn't related to this PR. Would you update the PR text to indicate why this is changing?

Reply:
this is only in a test, to have a field that can trigger nullability rules